lib/data_migrate/tasks/setup_tests.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module DataMigrate
+  module Tasks
+    class SetupTests
+      INJECTION_MATCHER = Regexp.new(/require_relative ["|']\.\.\/config\/environment["|']/)
+
+      def call
+        return if injection_exists?
+
+        if find_injection_location.nil?
+          puts 'data_migrate: config/environment.rb was not found in the test helper file.'
+          return
+        end
+
+        add_newline
+
+        lines_for_injection.reverse.each do |line|
+          file_contents.insert(find_injection_location, "#{line}\n")
+        end
+
+        add_newline
+
+        File.open(test_helper_file_path, 'w') do |file|
+          file.puts file_contents
+        end
+
+        puts 'data_migrate: Test setup complete.'
+      end
+
+      private
+
+      def test_helper_file_path
+        case DataMigrate::Helpers::InferTestSuiteType.new.call
+        when :rspec
+          Rails.root.join('spec', 'rails_helper.rb')
```

Comment:
`rails_helper.rb` contains the injection target BUT DataMigrate::Helpers::InferTestSuiteType checks `spec_helper.rb` (see: https://github.com/ilyakatz/data-migrate/pull/355/files#diff-e3eb6de888eae80ac87faa33be0419ff8abdc2bd7d14e23c243304834d53604eR5). I'm thinking these maybe should match in their conditional checks. Thoughts?